xline/src/storage/auth_store/store.rs

```diff
@@ -39,40 +84,791 @@ where
         key_pair: Option<(EncodingKey, DecodingKey)>,
         header_gen: Arc<HeaderGenerator>,
         storage: Arc<S>,
-    ) -> Self {
-        Self {
-            inner: Arc::new(AuthStoreBackend::new(
-                lease_cmd_tx,
-                key_pair,
-                header_gen,
-                storage,
-            )),
+    ) -> Result<Self, ExecuteError> {
+        let backend = Arc::new(AuthStoreBackend::new(storage));
+        let revision = backend.get_revision()?;
+        let enabled = backend.get_enable()?;
```

Comment:
just use default values, I will implement recover logic in another method

Reply:
Fixed